src/domain/entities/Document.ts

```diff
@@ -0,0 +1,4 @@
+export interface Document {
```

Comment:
Last year we started to move to declare entities that will not be implemented as `type = { ... }` instead of `interface`